src/app/containers/Header/ScriptLink/index.jsx

```diff
@@ -36,7 +43,11 @@ const ScriptLinkContainer = () => {
         variant,
       })}
       variant={variant}
-      onClick={() => setPreferredVariantCookie(service, variant)}
+      onClick={() => {
+        return (
+          variantCookieEnabled && setPreferredVariantCookie(service, variant)
+        );
+      }}
```

Comment:
```suggestion
      onClick={() => (
        variantCookieEnabled && setPreferredVariantCookie(service, variant)
      )}
```
You can use an implicit return here :)

Reply:
I have tried doing this, but EsLint is complaining about it

Reply:
![image](https://user-images.githubusercontent.com/30599794/76220191-fa805500-620e-11ea-9302-591a988e4550.png)
😭

Reply:
Since the `variantCookieEnabled` toggle is couple to the setPreferredVariantCookie, maybe we could move the enabled check into where the `setPreferredVariantCookie` is defined (`src/app/contexts/UserContext/cookies/index.js`) and update the tests there.

Reply:
Will try and do that 👍

Reply:
My suggestion was actually invalid, as `setPreferredVariantCookie` is not a react component, we won't be able to use the toggleContext inside of the function.